pyintegration/deephaven2/hmtl.py

```diff
@@ -0,0 +1,24 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+import jpy
+
+from deephaven2 import DHError
+from deephaven2.table import Table
+
+_JTableTools = jpy.get_type("io.deephaven.engine.util.TableTools")
+
+
+def to_html(table: Table) -> str:
+    """  Returns a printout of a table formatted as HTML. Limit use to small tables to avoid running out of memory.
+
+    Returns:
+        a String of the table printout formatted as HTML
+
+    Raises:
+        DHError
+    """
```

Comment:
```suggestion
    """  Returns a table formatted as an HTML string. Limit use to small tables to avoid running out of memory.

    Returns:
        a table formatted as an HTML string

    Raises:
        DHError
    """
```